### ISSUE TITLE
setup.sh missing

### ISSUE THREAD

**Issue by ghost** [flourgaz/invidious-docker#1]
The [readme](https://github.com/flourgaz/invidious-docker/blob/master/README.md) say to run setup.sh, but this file is missing or no longer part of Invidious.

   - inside the container, run setup

     `cd /invidious; ./setup.sh`

Is there another way to prepare the `invidious_postgresdata` volume?

**Comment by unixfox** [comment]
Yes you can just populate the database with `psql` like described in the README: https://github.com/omarroth/invidious#setup-postgressql
Just be aware that it's a little bit different with docker so this commit will most likely help you for modifying the command for Docker: https://github.com/omarroth/invidious/issues/926#issuecomment-559199910
